packages/cli/src/dbt/models.ts

```diff
@@ -172,7 +173,7 @@ export const findAndUpdateModelYaml = async ({
         filenames.push(expectedYamlPath);
     }
     const defaultYmlPath = path.join(
-        path.dirname(path.join(projectDir, model.originalFilePath)),
```

Comment:
are we making assumption the package name equals the folder name?

Reply:
I'm assuming they are the same, I don't know if it's possible for them to differ